themes/containerd/layouts/partials/footer.html

```diff
@@ -7,19 +7,19 @@
 {{- $githubUrl     := printf "https://github.com/%s" $githubRepo }}
 {{- $slackChannel  := .Site.Params.slackChannel }}
 {{- $projectPages  := where .Site.RegularPages "Section" "" }}
-<div class="footer">
+<div class="footer has-background-pattern has-text-white">
   <div class="container">
     <div class="columns">
-      <div class="column">
-        <img src="{{ $logo }}" class="is-footer-logo" alt="containerd footer logo">
+      <div class="column" style="display: flex;justify-content: center;align-items: center;">
+        <img style=" transform:scale(1.2);" src="{{ $logo }}" class="is-footer-logo" alt="containerd footer logo">
       </div>
       
        <div class="column">
-        <span class="is-size-4 is-size-5-mobile">
+        <span class="is-size-4 is-size-5-mobile has-text-weight-bold">
           Docs
         </span>
 
-        <hr class="hr has-background-grey">
+        <hr class="hr  ">
```

Comment:
Nit: remove unnecessary space in class string, here and the other instances below.